HelpSource/Classes/SimpleNumber.schelp

```diff
@@ -368,16 +368,16 @@ method:: asAudioRateInput
 Converts this into an audiorate input.
 
 method:: asTimeString
-Produces a time string in the format code::ddd:hh:mm:ss.sss::, interpreting the receiver as time in
-seconds. See link::Classes/String#-asSecs:: for the inverse function.
+Produces a time string in the clock format inspired by ISO 8601 time interval display (truncated representation) code::ddd:hh:mm:ss.sss::, interpreting the receiver as time in seconds. See link::Classes/String#-asSecs:: for the inverse function.
```

Comment:
maybe good to note here that the exact format is configurable, and not fixed at `ddd:hh:mm:ss.sss`